src/pkg/retention/policy/rule/lastx/evaluator_test.go

```diff
@@ -0,0 +1,69 @@
+package lastx
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/goharbor/harbor/src/pkg/retention/policy/rule"
+	"github.com/goharbor/harbor/src/pkg/retention/res"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEvaluator_New(t *testing.T) {
```

Comment:
Let's drive the test cases with the testing framework `github.com/stretchr/testify` from now on.

Reply:
Is there a proposal somewhere pitching this? I'm fine changing it out but it feels like a lot of added overhead for simple tests like this.

Reply:
@nlowe 

No proposal about changing UT approach. The reason I'd like to recommend it because we have introduced that framework a long time ago and some UT cases have implemented with it. The previous cases will be kept as they're. We'd like to apply a unified way for the newly introduced cases. Obviously, testify is better than the native test because more libraries are provided and well-formated results are supported.

I need to admit that this is a recommended approach, not mandatory as we did not raise the proposal to discuss it (actually I'm doubting that does it really need a formal proposal to discuss). If you insist on the native case way, I'm fine with that.